src/iris/api.py

```diff
@@ -5043,17 +5044,22 @@ def on_get(self, req, resp, username):
         cursor.close()
         conn.close()
         if user_email is None:
-            raise HTTPBadRequest('User does not exist in Iris')
+            resp.body = 'User does not exist in Iris'
+            raise HTTPBadRequest()
 
         # query slack for user id from email address
         slack_vendor = iris_slack(self.slack_cfg)
-        slack_id = slack_vendor.lookup_by_email(user_email['destination'])
+        try:
+            slack_id = slack_vendor.lookup_by_email(user_email['destination'])
+        except Exception as e:
+            raise HTTPInternalServerError()
```

Comment:
Can you also send the exception detail as part of the response?

Reply:
sure